packages/apps/crm/src/i18n/en.json

```diff
@@ -118,5 +118,6 @@
   "Crm_SliceAction_CreateOpportunity": "create opportunity",
   "Crm_SliceAction_FetchCompanyById": "get company by id",
   "Crm_SliceAction_CreateCatalog": "Create catalog",
-  "Crm_SliceAction_FetchProspectStatus": "fetch prospect status"
+  "Crm_SliceAction_FetchProspectStatus": "fetch prospect status",
+  "Crm_SaveContact": "Save contact"
```

Comment:
We try to keep translation files readable even with all the slice translations, so please follow the existing structure : 
first part of file should contain the normal translation that are displayed to user and the end of the file should contain the slice translations.

You translation should be above the SliceAction. 
Please modify fr and en files.